lua/ttt2/extensions/util.lua

```diff
@@ -150,6 +150,17 @@ function util.GetNextAlivePlayer(ply)
 	return alive[1]
 end
 
+---
+-- Returns the list of active @{Player}s that are not forced spectators
+-- @return table List of active @{Player}s
+-- @realm shared
+function util.GetActivePlayers()
+	return util.GetFilteredPlayers(
+		function(ply)
+			return IsValid(ply) and not ply:GetForceSpec()
+		end)
+end
```

Comment:
imho this should be:
```suggestion
function util.GetActivePlayers()
	return util.GetFilteredPlayers(function(ply)
		return IsValid(ply) and not ply:GetForceSpec()
	end)
end
```

or:
```suggestion
function util.GetActivePlayers()
	return util.GetFilteredPlayers(
		function(ply)
			return IsValid(ply) and not ply:GetForceSpec()
		end
	)
end
```